### PR TITLE
little cleanup

### PR DESCRIPTION
**Pull request by retokromer**
Should improve readability of the code for maintenance:
- uniform alignment of playback filters
- uniform use of `elif` instead of `fi ; if` inside the same logical structure

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -308,7 +308,7 @@ duration_check(){
             _report -w "Illegal value for recording time. Input must only be numbers."
             exit 1
         fi
-        if [[ 1 -eq "$(echo "${duration} == 0" | bc)" ]] ; then
+        if [[ 1 -eq "$(echo "${duration} = 0" | bc)" ]] ; then
```
Comment (dericed) [inline vrecord]: duration="0.000" ; echo "${duration} = 0" | bc
(standard_in) 1: parse error

Comment (retokromer) [reply vrecord]: Thank you, I’ll fix! (The behaviour is not clear to me from the top of the head.)

Comment (dericed) [reply vrecord]: see https://www.gnu.org/software/bc/manual/html_mono/bc.html#SEC11

Comment (retokromer) [reply vrecord]: Ah, I understand, it’s not the double [[. – Thank you very much, as usual!

**Pull request merged**